cli/medperf/commands/auth/login.py

```diff
@@ -1,8 +1,8 @@
 import os
 import stat
+import configparser
 
 import medperf.config as config
-from medperf.utils import storage_path
 
 
 class Login:
```

Comment:
I wonder though how the client integration test check passes

Reply:
I believe this is happening because you still have a `credentials` file from previous versions, which stored things differently. This is a breaking change, so it's best to try it on a clean medperf environment

Reply:
True :)